Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/sources/remote/RemoteAppConfigSource.kt

```diff
@@ -48,7 +48,7 @@ class RemoteAppConfigSource @Inject constructor(
     }
 
     fun clear() {
-        server.clearCache()
+        Timber.tag(TAG).v("clear()")
```

Comment:
`clear` now  not really  clearing either to change the name or document why it has to be like that